test-prow-e2e.sh

```diff
@@ -66,7 +66,7 @@ export DBUS_SESSION_BUS_ADDRESS
 
 SCENARIO="${1:-e2e}"
 
-if [ "$SCENARIO" != "login" ] && [ "$SCENARIO" != "olmFull" ] && [ "$SCENARIO" != "ceph" ]; then
+if [ "$SCENARIO" != "login" ] && [ "$SCENARIO" != "olmFull" ] && [ "$SCENARIO" != "ceph" ] && [ "$SCENARIO" != "kubevirt-gating" ]; then
```

Comment:
There are enough items now that we should try to clean up this if statement. My bash isn't great, but I *think* this will work. (Please test.)

```bash
case $SCENARIO in
  login|olmFull|ceph|kubevirt-gating) ;; # no protractor tests
  *) CHROME_VERSION=$(google-chrome --version) ./test-protractor.sh "$SCENARIO";;
esac
```

Reply:
``` bash
if (echo "login olmFull ceph kubevirt-gating" | grep -qvw "$SCENARIO") ; then
        CHROME_VERSION=$(google-chrome --version) ./test-protractor.sh "$SCENARIO"
fi

```
may also work